README.md

```diff
@@ -714,6 +714,17 @@ Browsershot::url('https://example.com')
    ...
 ```
 
+#### Sending POST requests
+
+By default, all requests sent using GET method. You can make POST request to the given url by using the `postParams` method.
+Note: POST request sent using `application/x-www-form-urlencoded` content type.
+
+```php
+Browsershot::url('https://example.com')
+    ->postParams(['foo' => 'bar'])
```

Comment:
Let's use the method `post` which is a bit friendlier. Default argument of the function should be an empty array.